Assets/MXR.SDK/Runtime/Android/MXRAndroidSystem.cs

```diff
@@ -168,14 +223,23 @@ public class MXRAndroidSystem : IMXRSystem {
                 messenger.Native?.Call<bool>("disableKioskModeAsync");
         }
 
-        public void ExitLauncher() {
+        public void Sync() {
             if (messenger.IsBoundToService)
-                messenger.Native?.Call<bool>("exitLauncherAsync");
+                messenger.Native?.Call<bool>("checkDbAsync");
         }
 
-        public void Sync() {
+        public void SendHomeScreenState(HomeScreenState state) {
+            if (messenger.IsBoundToService) {
+                Debug.Log("SendHomeScreenState called.");
+                // TODO: This might look something like this
```

Comment:
What else do we need to do to get this contract in a defined state so you can start sending it from the home screen?

Reply:
Got it, makes sense. Let's leave this as is then until we're ready from the android side. OK to merge now IMO.